src/components/charts-generic/histogram/histogram.stories.tsx

```diff
@@ -76,3 +80,75 @@ export const Histogram: Story = {
     );
   },
 };
+
+export const GroupedHistogram: Story = {
+  render: () => {
+    // Use operatorId from the imported JSON for annotation
+    const operatorId =
+      operationalStandards.serviceQuality.operatorsNotificationPeriodDays[0]
+        .operatorId;
+    const operatorLabel = `Operator ${operatorId}`;
+    // Create mock grouped data (like operational-standards-chart.tsx)
+    const mockGroupedData = Array.from({ length: 30 }, (_, i) => ({
+      id: i === 0 ? operatorId : Math.random() * 100,
+      value: Math.round(Math.random() * 30),
+      label: `Operator ${i + 1}`,
+    }));
```

Comment:
The fact that there is randomness here will make this story not testable. I think the mock data should be saved and directly passed instead of being calculated.